tests/kafkatest/version.py

```diff
@@ -249,7 +250,3 @@ def get_version(node=None):
 # 3.5.x versions
 V_3_5_0 = KafkaVersion("3.5.0")
 LATEST_3_5 = V_3_5_0
-
-# 3.6.x versions
-V_3_6_0 = KafkaVersion("3.6.0")
```

Comment:
Here -- did we mean to remove?

Reply:
guessing it was removed because there were no references to it, I'll add it back when I raise a PR for more comprehensive upgrade tests

Reply:
Yes -- 3.6.0 is not released yet. We should only need it after 3.6.0 is out.

If you want to refer to current version it would be `DEV_VERSION` (ie, 3.6.0-SNAPSHOT).

Reply:
Hmm -- if you look at all the snapshot commits we've added this upon releasing the previous version. (Ie, cutting the branch for the new trunk as per the commit comments)
https://github.com/apache/kafka/commit/dc1ede8d89d0964783302e0da9ead7fa1d76fbe4
https://github.com/apache/kafka/commit/c1a54671e8fc6c7daec5f5ec3d8c934be96b4989
https://github.com/apache/kafka/commit/6ace67b2de00f4f1665e7c9e3484e01b6d6f9584

Reply:
I think the issue in David's commit was that he didn't set the dev version to match the new version defined?

Reply:
Could be -- I mean, it does not really hurt to add it early (it's just a definition of a var), but it's also useless to have, a nobody should use it, but use `DEV_VERSION` instead.